core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -40,6 +40,7 @@ import org.apache.ivy.plugins.resolver.{ChainResolver, IBiblioResolver}
 import org.apache.spark.deploy.rest._
 import org.apache.spark.executor._
 import org.apache.spark.util.{ChildFirstURLClassLoader, MutableURLClassLoader, Utils}
+import org.apache.spark._
```

Comment:
I would just explicitly import `SPARK_VERSION` since that's the only thing we need here